packages/core/js-sdk/src/store/index.ts

```diff
@@ -941,6 +941,38 @@ export class Store {
         }
       )
     },
+
+    /**
+     * This method updates the customer of a cart.
+     *
+     * @param id - The cart's ID.
+     * @param body - The data to update in the cart.
+     * @param query - Configure the fields to retrieve in the cart.
+     * @param headers - Headers to pass in the request.
+     * @returns The updated cart.
+     *
+     * @example
+     * sdk.store.cart.updateCustomer("cart_123", {})
+     * .then(({ cart }) => {
+     *   console.log(cart)
+     * })
+     */
+    updateCustomer: async (
+      id: string,
+      body: HttpTypes.StoreUpdateCartCustomer,
```

Comment:
thought: Can we remove the `body` param and always pass `{}` in the `fetch` call?

That way you could consume it like so:
```ts
sdk.store.cart.transferCustomer("cart_123")
```